commands/randomList.js

```diff
@@ -0,0 +1,28 @@
+const {SlashCommandBuilder, EmbedBuilder} = require('discord.js');
+const API = require('../Helpers/JsonServer.js');
+const crypto = require("crypto");
+
+const randomList = async interaction => {
+	await interaction.reply(`Fetching your random deck list...`);
+	const deckLists = await API.getAllDecks();

```

Comment:
should this be done at the API level? also, would it be more efficient to choose the deck at the db/cache layer instead of in JS?